From 658ce8cd13ac5d672f2e3e6bcabe324fe293e54b Mon Sep 17 00:00:00 2001
From: radhit <radhit@iitk.ac.in>
Date: Mon, 5 Mar 2018 00:52:30 +0530
Subject: [PATCH] Modified Version-8 of hello.c by user_2 using branch master.

---
 hello.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hello.c b/hello.c
index 152ebd8..8964913 100644
--- a/hello.c
+++ b/hello.c
@@ -5,6 +5,7 @@ void microkernel_sendmsg(char *);
 void main(){
 printf("Helloworld!\n");
 printf("This must be a monolithic design\n");
+printf("Assignment-5 is over\n");
 microkernel_sendmsg("is more portable");
 }
 
-- 
2.7.4

